addons/mail/static/tests/systray/systray_messaging_menu_tests.js

```diff
@@ -1072,11 +1068,10 @@ QUnit.test('messaging menu widget: click twice preview on slow message_fetch sho
             if (args.method === 'channel_minimize') {
                 // called to detach thread in chat window
                 // simulate longpolling response with new chat window state
-                const channelInfo = {
-                    ...self.data['mail.channel'].records[0],
+                const channelInfo = Object.assign({}, self.data['mail.channel'].records[0],{
                     is_minimized: true,
                     state: 'open',
-                };
+                });
```

Comment:
wouldn't make this change though. Chromium edge is being automatically deployed since almost 2 months now, and this syntax is nicer. Also this is code in test, so I would keep using spread operator on object.

Reply:
Can't lint the rest of the file if there is a syntax error on it. Need to be consistent until we can actually change the rule everywhere.